app/src/ui/app.tsx

```diff
@@ -348,11 +348,64 @@ export class App extends React.Component<IAppProps, IAppState> {
         return this.openCurrentRepositoryInExternalEditor()
       case 'select-all':
         return this.selectAll()
+      case 'show-release-notes-popup':
+        return this.showFakeReleaseNotesPopup()
     }
 
     return assertNever(name, `Unknown menu event name: ${name}`)
   }
 
+  /**
+   * Show a release notes popup for a fake release, intended only to
+   * make it easier to verify changes to the popup. Has no meaning
+   * about a new release being available.
+   */
+  private showFakeReleaseNotesPopup() {
+    this.props.dispatcher.showPopup({
```

Comment:
@koralcem actually, we talked about wrapping this in a `if(__DEV__)` so it would get excluded from the production build. Would you mind adding this in before I merge it?

Reply:
Wow... Good catch @shiftkey, and I'm so sorry I forgot about that. As you said, we just talked about it too 🤦‍♂️ 

Added.